lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -3010,7 +3010,9 @@ void mme_app_handle_erab_rel_cmd(
           (rel_index < pdn_context_p->esm_data.n_bearers));
          idx++) {
       int8_t bearer_index = pdn_context_p->bearer_contexts[idx];
-      if (ue_context_p->bearer_contexts[bearer_index]) {
+      if (
+        (bearer_index != INVALID_BEARER_INDEX) &&
```

Comment:
It will be safer to use INVALID_BEARER_INDEX in everywhere the bearer context is initialized. Currently INVALID_BEARER_INDEX is only used here, while initialization is done somewhere else by hardcoding to -1.  If someone modifies that part of the code (e.g., initialized to something else), the logic here will be false.